src/redux/books/books.js

```diff
@@ -1,36 +1,39 @@
 // Actions
-const ADD_BOOK = 'React-BookStore/books/ADD_BOOK';
-const REMOVE_BOOK = 'React-BookStore/books/REMOVE_BOOK';
+const ADD_BOOK = 'bookstore-app/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore-app/books/REMOVE_BOOK';
 
-// Reducer
-const addremoveReducer = (state = [], action) => {
-  switch (action.type) {
-    // do reducer stuff
-    case ADD_BOOK: {
-      return [...state, {
-        id: action.id,
-        title: action.title,
-        author: action.author,
-      }];
-    }
-    case REMOVE_BOOK: {
-      return state.filter((book) => book.id !== action.id);
-    }
-    default:
-      return state;
-  }
-};
-// Action Creators
-export const addBook = (id, title, author) => ({
+const initialState = [
+  {
+    id: 1,
+    title: '100 WAYS TO BE MONEY WISER',
+    author: 'Tilda Nabbanja',
+  },
+  {
+    id: 2,
+    title: 'BACK HOUSE',
+    author: 'Butamanya George',
+  },
+];
+
+export const addBook = (book) => ({
   type: ADD_BOOK,
-  id,
-  title,
-  author,
+  payload: book,
 });
 
-export const removeBook = (id) => ({
+export const removeBook = (index) => ({
   type: REMOVE_BOOK,
-  id,
+  payload: index,
 });
 
-export default addremoveReducer;
+const reduceBook = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return [...state.filter((item) => item.id !== action.payload)];
```

Comment:
[Optional] The array filter method returns a new array and so, using the spread operator to avoid mutation was not necessary since the same array is not returned.

Something like this will work:
`state.filter((item) => item.id !== action.payload)`